docs/docs/deploy/fly.md

```diff
@@ -0,0 +1,12 @@
+# Deploy to Fly.io
+
+
+Fly.io allows you to run your full stack apps (and databases!) all over the world. No ops required. A simple Redwood app can run permanently (no time limits) on the Fly free tier with a 1GB Postgresql database, or a 1GB dedicated storage volume for SQLite.
+
+## FLy.io tl;dr Deploy
+
+If you simply want to experience the Fly.io deployment process, including a Postgres or SQLite database, you can do the following:
```

Comment:
Before and above you used Postgresql as name for PG, so i assume "Postgres" is a typo.